app/src/main/java/com/dlewburg/taskmanagerandroid/adapters/TaskListRecyclerViewAdapter.java

```diff
@@ -0,0 +1,78 @@
+package com.dlewburg.taskmanagerandroid.adapters;
+
+import static com.dlewburg.taskmanagerandroid.MainActivity.TASK_DESCRIPTION_TAG;
+import static com.dlewburg.taskmanagerandroid.MainActivity.TASK_DETAILS_TITLE_TAG;
+import static com.dlewburg.taskmanagerandroid.MainActivity.TASK_STATUS_TAG;
+
+import android.content.Context;
+import android.content.Intent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.dlewburg.taskmanagerandroid.R;
+import com.dlewburg.taskmanagerandroid.activities.TaskDetailsActivity;
+import com.dlewburg.taskmanagerandroid.models.Task;
+
+import java.util.List;
+
+public class TaskListRecyclerViewAdapter extends RecyclerView.Adapter<TaskListRecyclerViewAdapter.TaskListViewHolder> {
+
+    private List<Task> tasks;
+    Context callingActivity;
+
+    public TaskListRecyclerViewAdapter(List<Task> tasks, Context callingActivity) {
+        this.tasks = tasks;
+        this.callingActivity = callingActivity;
+    }
+
+    @NonNull
+    @Override
+    public TaskListViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View taskFragment = LayoutInflater.from(parent.getContext()).inflate(R.layout.fragment_task_list, parent, false);
+        return new TaskListViewHolder(taskFragment);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull TaskListViewHolder holder, int position) {
+        Button taskFragmentButton = (Button) holder.itemView.findViewById(R.id.fragmentTaskListButton);
+        Task task = tasks.get(position);
+        String taskName = task.getTitle();
+        taskFragmentButton.setText(taskName);
+
+        setupTaskButton(taskFragmentButton, task);
+    }
+
+    @Override
+    public int getItemCount() {
+        return 0;
```

Comment:
This is where the recyclerView checks how many items it needs to display. Since it always returns 0, nothing will display; instead, it needs to return the size of the task list. 

To troubleshoot it, I first tried a bunch of layout tweaks; when that didn't work, I googled "android recyclerview not showing" and luckily, the first comment on [this post](https://stackoverflow.com/questions/27705928/recycler-view-not-showing) had the answer!